main.rb

```diff
@@ -0,0 +1,7 @@
+require './App'
```

Comment:
hi @FrederickMih From my end it was working fine. Whenever I run rubocop linters it does away with the .rb extension. 

![Screenshot (78)](https://user-images.githubusercontent.com/62090797/180262190-0818034b-a26b-4457-aff4-1aeff92d562f.png)
.
![Screenshot (79)](https://user-images.githubusercontent.com/62090797/180262908-5b7dff13-aa82-4cb7-9a17-c727604c9f7a.png)

Reply:
@kwambiee, the problem was not the file extension but the file name. You were previously using uppercase **A** instead of lowercase **a**. You can check once more the last commit for the previous PR.